### DISCUSSION TITLE
Support uWebSockets

### DISCUSSION THREAD

**Discussion by rtritto** [vikejs/bati#408] [Integration request]
### Name

uWebSockets

### Homepage

https://github.com/uNetworking/uWebSockets.js

### Additions
Benchmarks:
- https://github.com/uNetworking/uWebSockets/tree/master/benchmarks
- https://github.com/SaltyAom/bun-http-framework-benchmark#results
- https://web-frameworks-benchmark.netlify.app/result?l=javascript
- https://www.techempower.com/benchmarks/#hw=ph&test=fortune&section=data-r22&l=zik0sf-cn3

**Comment by magne4000** [comment]
How would you see it integrated into Bati?

**Comment by rtritto** [comment]
It can be used as any Server in Bati.

Migration from Express: https://dev.to/mattkrick/replacing-express-with-uwebsockets-48ph
Migration from Koa: https://dev.to/atonchev/migrating-from-koa-to-uwebsocketsjs-in-less-than-2-hours-51h3

---
Some examples (use cases): https://github.com/uNetworking/uWebSockets.js/tree/master/examples

Maybe it can also be added in other sections instead of just the Sever one.
